### PR TITLE
Add detection sensor range

### PR DESCRIPTION
**Pull request by kyoichi-sugahara**
## Types of PR

- [x] New Features
- [ ] Upgrade of existing features
- [ ] Bugfix

## Description

Add sendor deteciton range which can be set in the scenario file with the parameter of detectionSensorRange.

https://github.com/tier4/scenario_simulator_v2/assets/32741405/eff579b7-8707-4e40-bb1a-f668f0b04a40

- remarks
　- default value is 300
　- parameter of `isClairvoyant` should be set as true explicitly in the scenario file

(I will add detail later)

### CODE REVIEW

**Review by HansRobo — CHANGES_REQUESTED** [review]
The Documentation is required

File: docs/developer_guide/ConfiguringPerceptionTopics.md [hunk 1]
```diff
@@ -22,12 +22,14 @@ Perception topics are configurable with the following syntax:
...
+| `randomSeed`                              | A positive `integer` type value               |   `0`   | Specifies the seed value for the random number generator.                                                                                                                                                            |
+| `detectionSensorRange`                    | A positive `double` type value                | `300.0` | Specifies the sensor detection range for detected object.                                                                                                                                                            |
+| `isClairvoyant`                           | A `boolean` type value                        | `false` | Specifies whether the detected object is a Clairvoyant. If this parameter is not defined explicitly, the property of `detectionSensorRange` is not reflected and only detected object detected by lidar is published |
```
Comment (hakuturu583) [inline docs/developer_guide/ConfiguringPerceptionTopics.md:32]: The unmodified ones are listed in alphabetical order.

Comment (hakuturu583) [reply docs/developer_guide/ConfiguringPerceptionTopics.md:32]:
```suggestion
| Name                                      | Value                                         | Default | Description                                                                                                                                                                                                          |
| ----------------------------------------- | --------------------------------------------- | :-----: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `detectedObjectMissingProbability`        | A `double` type value between `0.0` and `1.0` |  `0.0`  | Do not publish the perception topic with the given probability.                                                                                                                                                      |
| `detectedObjectPositionStandardDeviation` | A positive `double` type value                |  `0.0`  | Randomize the positions of other vehicles included in the perception topic according to the given standard deviation.                                                                                                |
| `detectedObjectPublishingDelay`           | A positive `double` type value                |  `0.0`  | Delays the publication of the perception topic by the specified number of seconds.                                                                                                                                   |
| `detectionSensorRange`                    | A positive `double` type value                | `300.0` | Specifies the sensor detection range for detected object.                                                                                                                                                            |
| `isClairvoyant`                           | A `boolean` type value                        | `false` | Specifies whether the detected object is a Clairvoyant. If this parameter is not defined explicitly, the property of `detectionSensorRange` is not reflected and only detected object detected by lidar is published |
| `randomSeed`                              | A positive `integer` type value               |   `0`   | Specifies the seed value for the random number generator.                                                                                                                                                            |
```

**Comment by kyoichi-sugahara** [comment]
close this PR and make new PR with the new design

**Pull request closed without merging**